templates/admin/activity/show.html.twig

```diff
@@ -2,12 +2,6 @@
 
 {% if page_title is not defined %}
     {% set page_title = 'Activiteit ' ~ activity.name %}
-    
-    {% block buttons %}
-        <a href="{{ path('admin_activity_image', { 'id': activity.id }) }}">Afbeelding aanpassen</a>
-        <a href="{{ path('admin_activity_edit', { 'id': activity.id }) }}">Bewerken</a>
-        <a href="{{ path('admin_activity_delete', { 'id': activity.id }) }}">Verwijderen</a>
```

Comment:
Activities should be deletable somehow, how can this be accomplished now?

Reply:
they shouldn't. As discussed during our meeting this PR aims to just enable archiving. I do agree that my first commit msg doesn't reflect this. Once you're satisfied with this PR I'll rebase to edit them accordingly.